blssig/cache_test.go

```diff
@@ -0,0 +1,51 @@
+package blssig
+
+import (
+	"runtime"
+	"testing"
+
+	bls12381 "github.com/drand/kyber-bls12381"
+	"github.com/drand/kyber/sign/bdn"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-f3/gpbft"
+)
+
+const maxCacheMemory uint64 = 10 * 1023 * 1024 // 10MiB
+
+func TestCacheMemory(t *testing.T) {
```

Comment:
While at it assert max pub key size too?

Reply:
Huh. The answer is actually 48. I think it's 96 for _signatures_?

Reply:
Fixed and https://github.com/filecoin-project/go-f3/issues/547